app/views/members/_member_form_non_impaired.html.erb

```diff
@@ -29,33 +29,38 @@ See doc/COPYRIGHT.rdoc for more details.
 
 <%= javascript_include_tag "members_select_boxes.js" %>
 
-<%= form_for(:member, :url => {:controller => 'members', :action => 'create', :project_id => project},
+<%= labelled_tabular_form_for(:member, :url => {:controller => 'members', :action => 'create', :project_id => project},
                      :remote => true,
                      :method => :post,
                      :html => {:id => "members_add_form"}) do |f| %>
-  <fieldset>
-    <legend><%= l(:label_member_new) %></legend>
-    <div class="select-boxes">
-      <label>
-        <%= l(:label_principal_search) %>
-        <%= select_tag "member[user_ids]", options_for_select([]),
-                         :title => l(:label_principal_search),
-                         :multiple => true,
-                         :'data-ajaxURL' => url_for(:controller => "/members", :action => "autocomplete_for_member"),
-                         :'data-projectId' => project.id,
-                         :class => "select2-select remote" %>
-      </label>
-      <label>
-        <%= l(:label_role_search) %>
-        <% options = roles.collect { |obj| [obj.name, obj.id] } %>
-        <%= select_tag 'member[role_ids]', options_for_select(options),
-                         :title => l(:label_role_search),
-                         :multiple => true,
-                         :tabIndex => 0,
-                         :class => "select2-select" %>
-      </label>
+  <div class="form--section">
+    <h3 class="form--section-title"><%= l(:label_member_new) %></h3>
+    <div class="grid-block">
+      <div class="form--column">
+        <div class="form--field -vertical">
+          <%= styled_label_tag :member_user_ids, l(:label_principal_search) %>
+          <%= styled_select_tag "member[user_ids]", options_for_select([]),
```

Comment:
@floriank I think this should be a regular old `select_tag`. Applying both `form--select` + select2 styling to an element is probably not desirable, and why it might (coincidentally) look close to the select2 mockups right now, it will probably break in the future.

![seeded_project_-_settings_-_openproject](https://cloud.githubusercontent.com/assets/755/6868124/0289f94a-d489-11e4-95e8-6d4117d2447b.png)